### DISCUSSION TITLE
关联视频一些地方不太人性化

### DISCUSSION THREAD

**Discussion by blibilijojo** [Richasy/Bili.Uwp#951] [常规]
如果关联视频自动播放的话按照的是本身的默认播放模式，导致我在全屏播放自动下一个的时候，下一个为默认播放
理想状态：在全屏下播放下一个，模式为全屏，默认模式播放下一个，模式为默认，其他同理

**Comment by Richasy** [comment]
可以在Issue提一个功能需求
